services/js/src/main/scala/org/apache/gearpump/dashboard/DashboardApp.scala

```diff
@@ -22,9 +22,7 @@ import scala.scalajs.js.JSApp
 import scala.scalajs.js.annotation.JSExport
 
 @JSExport
-object DashboardApp extends JSApp {
-  override def main(): Unit = {
-  }
+object DashboardApp {
```

Comment:
It is a stub for scalajs. Otherwise, the scalajs javascript will be not generated.